docs/2.deploy/20.providers/coolify.md

```diff
@@ -0,0 +1,23 @@
+# Coolify
+
+> Deploy Nitro apps to [Coolify](https://coolify.io).
+
+**Preset:** `coolify`
+
+:read-more{title="coolify.io" to="https://coolify.io"}
+
+::note
+This is the recommended preset for Coolify deployments.
+::
+
+::note
+Integration with this provider is possible with [zero configuration](/deploy#zero-config-providers).
+::
```

Comment:
[zero config](https://nitro.unjs.io/deploy#zero-config-providers) is explained in docs. It is for providers that nitro can **auto-detect their CI/CD** (with an environment variable), coolify is not this way.

Reply:
I removed the zero config provider lines accordingly.